azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/ProximityPlacementGroupImpl.java

```diff
@@ -69,12 +69,12 @@ public ProximityPlacementGroupInner inner() {
     }
 
 
-    private List<String> getStringListFromSubResourceList(List<SubResource> subList) {
+    private List<String> getStringListFromSubResourceList(List<SubResourceWithColocationStatus> subList) {
```

Comment:
What does this function do?

Reply:
It's to return a list of resource ids. The changes come from swagger and apply here.